### DISCUSSION TITLE
Is Azure renewed in Github Studen Developer Subscription?

### DISCUSSION THREAD

**Discussion by Souparno10** [community/community#33233] [GitHub Education]
I have recently claimed a Github Student Developer Pack and it's amazing..... but, I have noticed that it's written that some offers don't renew after the renewal of Github Student Developer pack. So, does the 100$ Azure package renew? I am asking because I use Azure the most.

**Comment by RonnieCarboney** [comment]
Currently

**Reply by Souparno10** [reply]
So, it does actually it means

**Reply by rbento01** [reply]
I think Ronnie has the correct idea

**Reply by rubenmuchaxo** [reply]
Generally you're right

**Comment by orkonung** [comment]
Hi. After the subscription expired, I was offered a new free subscription for a year with $100 credit

**Reply by Souparno10** [reply]
Thank You @orkonung, but can you mention when was it last renewed as the contents keep changing.
